### PR TITLE
Hello Microverse 

### PR DESCRIPTION
**Pull request by monika-soam**
- Created a new GitHub repo.   
- Created a new branch according to the [GitHub flow] rules.
- Added a .gitignore file.
- Set up linter for HTML and CSS
- Added a descriptive README file in my project 
- Add 3 files with a separate comment
- index.html file has a header with the text "Hello Microverse!".
- style.css file that includes one class with color style for header 
- test.md file with the text "This file should be ignored by git".

I am thankful in advance for being the reviewer of my first exercise.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules/
```
Comment (VICTOR-CHUKWU) [inline .gitignore]: - [ ] @monika-soam nice job in following the project requirement, Kindly add `text.md` file to your `.gitignore` file as required by the project

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @monika-soam  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (monika-soam) [reply .gitignore]: Sure, I will get this done in a few and resubmit.
Thank you

Comment (monika-soam) [reply .gitignore]: @VICTOR-CHUKWU . 
Changes are done, please check and let me know so I can proceed further.

**Review by Wes-Isaac — APPROVED** [review]
Hi @monika-soam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**